apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import { Stack, Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { useUser } from '@/helpers/user.helper';
+import { InfoCardItemProps } from '@/components/atoms/InfoCardItem/InfoCardItem';
+import { InfoCard } from '@/components/molecules/InfoCard/InfoCard';
+import PageNotFound from '@/app/not-found';
+
+const TermsOfService = () => {
+  const t = useTranslations('profile');
+  const { user, loading } = useUser();
+
+  const dataSectionItems: InfoCardItemProps[] = [
+    {
+      title: t('personalData.fields.name'),
+      value: user?.attributes.given_name,
+    },
+    {
+      title: t('personalData.fields.surname'),
+      value: user?.attributes.family_name,
+    },
+    {
+      title: t('personalData.fields.role'),
+      value: user?.attributes['custom:job_role'],
+    },
+    {
+      title: t('personalData.fields.company'),
+      // TODO: add company when available
+    },
+    {
+      title: t('personalData.fields.sector'),
+      value: user?.attributes['custom:company_type'],
+    },
+    {
+      title: t('personalData.fields.products'),
+      // TODO: add products when available
+    },
```

Comment:
```suggestion
```
At the moment I'd remove this. We don't even know yet how (nor when) the user could populate that information